### ISSUE TITLE
BackBlaze B2 Intergration with LGSM

### ISSUE THREAD

**Issue by dgibbs64** [GameServerManagers/LinuxGSM#1008]
I found this article recently and thought it would be a great idea to see if we can implement something like this. I already use B2 for my NAS and its the cheapest cloud backup solution around so I think it would be quite popular. 

http://ryanchristensen.net/how-to-backup-your-ark-server-to-b2-cloud-storage

**Comment by jach11** [comment]
Sounds like a good idea, maybe also expand on the current backup function to allow some improvements. It would be nice to introduce some automation for backups, so users can rsync their files to their backup servers, or even have archives to fallback on their local server.

**Comment by UltimateByte** [comment]
https://github.com/dgibbs64/linuxgsm/issues/716

Would be nice to get to some kind of a framework to upload, kinda like what you did for download recently. B2 or FTP, or whatever protocol. 
${what_to_upload}
${what_method}
${what_destination}
...

**Comment by cedarlug** [comment]
${what_creds}

**Comment by dgibbs64** [comment]
Yup i think improvements to the backup system would be create. I need a B2 backup script anyways. So I may release that for general server backups. However improving the existing LGSMbackup solution will be important to.

**Comment by MicLieg** [comment]
link #2106
